pages/common/bundletool.md

```diff
@@ -0,0 +1,36 @@
+# bundletool
+
+> Command-line tool to manipulate Android Application Bundles.
+> More information: <https://developer.android.com/studio/command-line/bundletool>.
+
+- Display help for a subcommand:
+
+`bundletool help {{subcommand}}`
+
+- Generate an APKs from an application bundle (prompts for keystore password):
+
+`bundletool build-apks --bundle={{path/to/bundle.aab}} --ks={{path/to/key.keystore}} --ks-key-alias={{key_alias}} --output={{path/to/file.apks}}`
+
+- Generate an APKs from an application bundle giving the keystore password:
+
+`bundletool build-apks --bundle={{path/to/bundle.aab}} --ks={{path/to/key.keystore}} --ks-key-alias={{key_alias}} –ks-pass={{pass:the_password}} --output={{path/to/file.apks}}`
+
+- Generate an APKs file including only one single APK for universal usage:
+
+`bundletool build-apks --bundle={{path/to/bundle.aab}} --mode={{universal}} --ks={{path/to/key.keystore}} --ks-key-alias={{key_alias}} --output={{path/to/file.apks}}`
+
+- Install the right combination of APKs to an emulator or device:
```

Comment:
```suggestion
- Install the default APK set archive to an emulator or device:
```

Reply:
Yes I wasn't sure about this one.

Reply:
The `.apks` has a lot of small `.apk` files inside, the Google's page says: `right combination of APKs`. That´s why `--mode-universal` is cool, because the `.apks` file only has 1 `.apk` file inside.

Reply:
Oh so in this case `APKs` is plural for `APK`?

Reply:
I replaces `APKs` with `APK set archive` because I thought that was what you meant, but did you mean plural of `APK`? :smile:

Reply:
oh! no, `APKs` is a specific `zip` with one or more `.apk` files inside.

Reply:
I reverted the suggestions then lol

Reply:
```suggestion
- Build an APK for the given emulator or device and install it:
```